mbta_finder.py

```diff
@@ -10,32 +10,88 @@
 from urllib.request import urlopen
 import json
 
+GoogleAPI = 'AIzaSyAox26uPgX2Gv-A0dNWiYN3D3ZuFQ6VKUg' 
```

Comment:
I'm thinking you don't want this public. Could you regenerate a new one and store it only on your local machine (steps can be found here https://sd17spring.github.io/day/day-9/), then remove this instance of it